### PR TITLE
Add Reviewboard Integration

### PR DESCRIPTION
**Pull request by JorisE**
#762

### CODE REVIEW

File: src/scripts/content/reviewboard.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*jslint indent: 2 */
+/*global $: false, document: false, togglbutton: false*/
+'use strict';
+
+togglbutton.render('#content_container:not(.toggl)', {observe: true}, function () {
```
Comment (langri-sha) [inline src/scripts/content/reviewboard.js]: This element appears in the root of all documents. Should select an element that's related to what's being tracked, perhaps `.review-request`?

Comment (JorisE) [reply src/scripts/content/reviewboard.js]: Yes, that would be an improvement.

**Comment by JorisE** [comment]
It works on my machine. Just to be sure, did you add the permission for reviewboard in the extension settings for toggl?

**Comment by JorisE** [comment]
The button should appear next to the summary. (Title of the review request).

**Comment by langri-sha** [comment]
Am I in the right view?

image

**Comment by JorisE** [comment]
You are :thinking: . Are you using the hosted version of reviewboard at rbcommons.com?

**Comment by JorisE** [comment]
toggl

This is what I have. Is there a repository for your reviewrequest? I thought that that is mandatory, but it may be possible to leave blank?

**Comment by JorisE** [comment]
@langri-sha: Reviewboard has rolled out a new version in which the toggl-button didn't work. Could you see if it works with the newest changes?

**Comment by JorisE** [comment]
There is an awaiting-response label on this PR. Is there anything else you expect me to change?

**Comment by JorisE** [comment]
@langri-sha I'd really like to start using the official releases of toggl-button again. Is there any way I can speed up this PR?

**Pull request merged**